go.mod

```diff
@@ -66,7 +66,7 @@ require (
 	github.com/icza/backscanner v0.0.0-20210726202459-ac2ffc679f94
 	github.com/influxdata/influxdb1-client v0.0.0-20200827194710-b269163b24ab
 	github.com/ipfs/bbloom v0.0.4
-	github.com/ipfs/go-bitswap v0.3.4
+	github.com/ipfs/go-bitswap v0.4.1-0.20211029155204-92d1e7aaf1dd
```

Comment:
Should use a tagged version

Reply:
blocked by the context thing

Reply:
blocked on the darn context data store change -- there is no tagged version between 0.3.4 and the context change that includes the independent updates needed (tied to update go-peertaskqueue)